src/components/Paginate.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useEffect, useState } from "react";
+
+const getPageCount = (
+  numberOfResults: number,
+  numberOfResultsPerPage: number
+): number => Math.ceil(numberOfResults / numberOfResultsPerPage);
+
+interface paginateProps {
```

Comment:
TS convention is for interfaces and types to be capital camelcase. use F2 in vscode to rename

Reply:
My mistake. f6d7450 fixes this. It's also the case in Java that interfaces start with a capital letter. I was so used to writing vanilla variable names that I forgot about the convention for interfaces.

Reply:
My mistake. 4205b74bc6927c6001c7bf482209ff7275c3a8a7 fixes this. It's also the case in Java that interfaces start with a capital letter. I was so used to writing vanilla variable names that I forgot about the convention for interfaces.